.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,65 @@
+name: Shellcheck Lint
+
+on:
+  push:
+    paths:
+      # Run workflow on every push
+      # only if a file within the specified paths has been changed:
+      - 'tests/**'
+      - 'usr/**'
+
+  pull_request:
+    paths:
+      # Run workflow on every push
+      # only if a file within the specified paths has been changed:
+      - 'tests/**'
+      - 'usr/**'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Shellcheck Lint
+
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+
+    steps:
+      # Required to access files of this repository
+      - uses: actions/checkout@master
```

Comment:
See https://github.com/rear/rear/pull/2756#discussion_r1048156831

Reply:
I agree and changed/pinned it to `v3`.